### ISSUE TITLE
WCAG 4.1.2 tests 4, 5: false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#97]
[This file](https://github.com/veraPDF/veraPDF-wcag-algs/files/7397499/test-document-6.pdf) fails WCAG 4.2.1 test 5 ('_Incorrect use of Hn tag_'). However, this error occurs on two lines of text which are correctly tagged as headings.

Note that the same false positive also occurs twice in [this other document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7397522/test-document-6.pdf)

Can you check out these false positives?

**Comment by jzuidweg** [comment]
That's a good one. But the error doesn't say they should be headings of different levels (which would be true), but that they should be paragraphs (which seems less logical). Taking the numbering into account seems like a good way to go.

If we want to be more sophisticated we could even introduce a new test which looks for 
1. headings of the same level that are formatted differently
2. headings of different levels that are formatted the same way (which is the case here)

and this could generate a warning of the kind 'confusing use of Hn tags' or something like that.

It's probably stricter than necessary for WCAG 2.1 but it could be a meaningful test.

**Comment by bdoubrov** [comment]
This seems to be a duplicate of #75.
